src/library/basesqltablemodel.cpp

```diff
@@ -265,18 +228,19 @@ void BaseSqlTableModel::select() {
     QVector<RowInfo> rowInfo;
     QSet<int> trackIds;
     while (query.next()) {
-        int id = query.value(idColumn).toInt();
+        int id = query.value(0).toInt();
```

Comment:
I'd prefer to make this a constant, kIdColumn, and just set it explicitly above.  it's too easy to change the query and break this literal number (even if, in this case, it's unlikely)